README.md

```diff
@@ -0,0 +1,31 @@
+# HTTP-Server[![Build Status](https://travis-ci.org/klkelley/http-java.svg?branch=master)](https://travis-ci.org/klkelley/http-java)
+
+
+## Using Docker 
```

Comment:
Can you throw some documentation in here about those 3 different tags you mentioned in standup?